src/Components/Indicators/ServerValidIndicator/ServerValidIndicator.jsx

```diff
@@ -45,7 +45,7 @@ const ServerValidIndicator = ({ setValid }) => {
         }
       })
       .finally(() => {
-        setIsLoading(false);
+        setTimeout(() => setIsLoading(false), 500);
```

Comment:
what is the purpose behind the timeout? If we want to leave this, we still have to put it into the `useEffect` cleanup function.

Reply:
The loading spinner and status strings are jumping without a timeout.

I've cleaned it. Please take a look.